content/en/releases/platforms.md

```diff
@@ -0,0 +1,336 @@
+periodic tests, watch them closely and raise the flag when things break. They
+will also have to help debug and fix any platform specific issues as well.
+
+Creating custom [testgrid][4] dashboards can help to monitor platform specific
+tests.
+
+[4]: https://testgrid.k8s.io
+
+#### Step 3: Releasing
+
+With the first 2 steps we have a reasonable expectation that there are people
+taking care of a supported platform and it works in a reproducible environment.
+
+Getting to the next level is a big jump from here. We are talking about real
+users who are betting their business literally on the work we are doing here. So
+we need guarantees around "can we really ship this!?" question.
+
+Specifically we are talking about a set of CI jobs in our release-informing and
+release-blocking tabs of our testgrid. The Kubernetes release team has a "CI
+signal" team that relies on the status(es) of these jobs to either ship or hold
+a release. Essentially, if things are mostly red with occasional green, it would
+be prudent to not even bother making this architecture as part of the release.
+CI jobs get added to release-informing first and when these get to a point where
+they work really well, then they get promoted to release-blocking.
+
+The problem here is once we start shipping something, users will start to rely
+on it, whether we like it or not. So it becomes a trust issue on this team that
+is talking care of a platform/architecture. Do we really trust this team not
+just for this release but on an ongoing basis. Do they show up consistently when
+things break, do they proactively work with testing/release on ongoing efforts
+and try to apply them to their architectures. It's very easy to setup a CI job
+as a one time thing, tick a box and advocate to get something added. It's a
+totally different ball game to be there consistently over time and show that you
+mean it. There has to be a consistent body of people working on this over time
+(life happens!).
+
+What are we looking for here, a strong green CI signal for release managers
+to cut a release and for folks to be able to report problems and them getting
+addressed. This includes [conformance testing][2] as use of the Kubernetes
+trademark is controlled through a conformance ensurance process. So we are
+looking for folks here to work with [the conformance sub project][3] in addition
+to testing and release.
+
+[2]: https://github.com/cncf/k8s-conformance
+[3]: https://bit.ly/sig-architecture-conformance
+
+#### Step 4: Finishing
+
+If you got this far, you really have made it! You have a clear engagement with
+the community, you are working seamlessly with all the relevant SIGs, you have
+your content in the Kubernetes release and get end users to adopt your
+architecture. Having achieved conformance, you will gain conditional use of the
+Kubernetes trademark relative to your offerings.
+
+#### Generic rules to consider
```

Comment:
Do we really need this list? I think it's very well covered by prior 4 points, and for such things, I don't think we should have tl;dr. 🙂

Reply:
I agree, let's remove it.